src/components/balanceCard.jsx

```diff
@@ -3,19 +3,25 @@ import { useParams } from 'react-router';
 import { Flex, Box, Image, useToast, Icon } from '@chakra-ui/react';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import { FaCopy } from 'react-icons/fa';
+import Web3 from 'web3';
 
 import { useDaoMember } from '../contexts/DaoMemberContext';
 import MinionTransfer from './minionTransfer';
 import SyncTokenButton from './syncTokenButton';
 import Withdraw from './withdraw';
 import { numberWithCommas } from '../utils/general';
 import { displayBalance } from '../utils/tokenValue';
+import { getWrapNZap } from '../utils/theGraph';
+import PokeTokenButton from './pokeTokenButton';
```

Comment:
import order. components above services and utils

https://daohaus.club/docs/devs/react-layount